nix/pkgs/urbit/default.nix

```diff
@@ -59,7 +59,7 @@ in stdenv.mkDerivation {
     then [ "--disable-shared" "--enable-static" ]
     else [];
 
-  CFLAGS = [ (if enableDebug then "-O0" else "-O3") "-g" ]
+  CFLAGS = (if enableDebug then [ "-O0" "-g" ] else [ "-O3" ])
```

Comment:
could this be made specific to arm64 since it doesn't appear to be an issue anywhere else? I assume we want debugging symbols in prod builds cc @joemfb @brendanhay

Reply:
Yeah, those symbols have been pretty handy on several occasions. I'd like to keep them around where possible.

Reply:
Done